LevelUp.Pos.ProposedOrders.Tests/Helpers/Check.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace LevelUp.Pos.ProposedOrders.Tests.Helpers
+{
+    /// <summary>
+    /// This class is meant to streamline expected calculations from a typical point of sale for the purpose of 
+    /// testing.  The behavior of this class expects a starting total amount due (including tax), and the tax amount.
+    /// It handles applying pre-tax discounts and standard payments/tenders.
+    /// </summary>
+    internal class Check
+    {
+        #region Private data members
+
+        private int StartingTotal { get; }
+        private int StartingTax { get; }
+        private int StartingSubtotal => StartingTotal - StartingTax;
```

Comment:
I think the common language in the restaurant world is subtotal.  How strongly do you feel about this?  Is there a reason "Subtotal" is not correct?